src/Services/OrderPropertyService.php

```diff
@@ -97,6 +97,17 @@ public function getCheckedPoNumber(int $orderId): string {
    */
   public function getWarehouseId(int $orderId): string {
     $orderProperties = $this->orderPropertyRepositoryContract->findByOrderId($orderId, OrderPropertyType::WAREHOUSE);
+    $this->loggerContract->debug(
+        TranslationHelper::getLoggerKey(self::LOG_KEY_WAREHOUSE_ID_NOT_FOUND),
+        [
+          'additionalInfo' => [
+            'orderId' => $orderId,
+            'typeId' => OrderPropertyType::WAREHOUSE,
+          'order properties' => $orderProperties
+          ],
+          'method' => __METHOD__
+        ]
+      );
```

Comment:
What is going on here? This is an unconditional log of `LOG_KEY_WAREHOUSE_ID_NOT_FOUND` **outside** of the conditional that checks for warehouse info.